wiremock/linkage/organizations.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "id": "MRI",
```

Comment:
We've gone with UK spelling for `organisations` elsewhere in the project, could we do that for this file? I think it's fine to pop `organisations.json` in the main `wiremock` directory, rather than nesting it in a `linkage` file too.

Reply:
So wiremock/, not wiremock/stubs/ ?

(Just to be clear, this file is not a stub, it's the linkage between some stubs which builds the referral dynamically).